src/components/Layout/navLinks.tsx

```diff
@@ -1,18 +1,18 @@
-import React from 'react'
+import { ReactNode } from 'react'
 
-import DiscordIcon from 'mdi-react/DiscordIcon'
 import GithubIcon from 'mdi-react/GithubIcon'
 import LinkedinIcon from 'mdi-react/LinkedinIcon'
 import SpotifyIcon from 'mdi-react/SpotifyIcon'
 import TwitterIcon from 'mdi-react/TwitterIcon'
 import YouTubeIcon from 'mdi-react/YoutubeIcon'
+import { FaDiscord as DiscordIcon } from 'react-icons/fa'
```

Comment:
We can eventually deprecate `mdi-react` in favour of `react-icons` which includes material design icons and many other libraries.